app/templates/admin.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="ember-admin">
   <h2>Admin</h2>
-  {{#each name in filteredModels}}
+  {{#each filteredModels key="@index" as |name|}}
```

Comment:
I don't think `key="@index"` is required anymore on the latest 1.13.x (I might be wrong)